packages/core/src/vite-plugin-astro-icon.ts

```diff
@@ -62,6 +62,7 @@ export function createPlugin(
           moduleGraph.invalidateAll();
         } catch (ex) {
           // Failed to load the local collection
+          throw new Error(`Failed to load local collection at: ${iconDir}`);
```

Comment:
Should this throw or should this remove the local collection then invalidate?

Reply:
If the error is thrown, it breaks the whole server, effectively stopping it from running until the configuration is reloaded.

Here are a couple of options:
- Report the error and continue
- Silently create the folder and continue
- Throw an error and stop the server until the error is fixed

The first two options resolve the issue, but trying to load a local icon will just break everything afterwards as the file will not exist.

Creating the folder may give a _slightly_ better experience though.